contrib/jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/IntegrationTest.groovy

```diff
@@ -85,15 +81,46 @@ class IntegrationTest extends Specification{
             jvmCommand.add("/app/${configName}")
         }
 
-        cassandraContainer =
-                new GenericContainer<>(
-                new ImageFromDockerfile().withFileFromString("Dockerfile", cassandraDockerfile))
-                .withNetwork(network)
-                .withNetworkAliases("cassandra")
-                .withExposedPorts(7199)
-                .withStartupTimeout(Duration.ofSeconds(120))
-                .waitingFor(Wait.forListeningPort())
-        cassandraContainer.start()
+        if ("cassandra" in targets) {
+            targetContainers.add(
+                    new GenericContainer<>(
```

Comment:
Might make things easier to read if this file were auto-formatted.

Reply:
Unfortunately this is how spotless formats groovy.  Moved the multiline string up for a bit of an improvement.